### ISSUE TITLE
What am I missing?  It doesn't convert math formula embedded at all. 

### ISSUE THREAD

**Issue by kundeng** [yzane/vscode-markdown-pdf#355]

**Comment by upyesp** [comment]
Thatis right, it doesn't support rendering math formula.  Therte's no suppoprt for Tex, Latex, Katex, MathJax.

If you want to convert Markdown with math formatting to PDF, try Pandoc.

https://pandoc.org/

**Comment by vinlin24** [comment]
See #21. You can append this `<script>` to your Markdown document. When it exports, the LaTeX should be rendered properly.

```md
<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({ tex2jax: {inlineMath: [['$', '$']]}, messageStyle: "none" });
</script>
```

**Comment by ruiyuanlu** [comment]
> See #21. You can append this `<script>` to your Markdown document. When it exports, the LaTeX should be rendered properly.
> 
> ```md
> <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
> <script type="text/x-mathjax-config">
>   MathJax.Hub.Config({ tex2jax: {inlineMath: [['$', '$']]}, messageStyle: "none" });
> </script>
> ```

Nice fix thx. But it does work in private net. Any offline alternatives available?

**Comment by yzane** [comment]
Built-in math rendering was added in **v2.1.0**. The earlier comments stating that Markdown PDF did not support math and required Pandoc or a remote MathJax script are now outdated.

Markdown PDF renders math with its bundled KaTeX integration during conversion. The required KaTeX CSS and fonts are included in the generated output, so math rendering does not require access to a CDN or other external network service. This also works in an offline or restricted private-network environment.

Closing the original lack-of-support request as resolved. If math still fails on v2.1.0 or later, please leave a minimal example and the output from `Markdown PDF: Output Diagnostics` — we will reopen and investigate.
